translations/de/hints/bryukh.html

```diff
@@ -0,0 +1,41 @@
+<body data-author="bryukh">
+<div class="question-answer">
+    <div class="question">
+        Ich habe keine Idee, wie ich anfangen soll, diese Mission zu lösen
+    </div>
+    <div class="answer">
+        Zuerst, lies über die Funktionen <a href="http://docs.python.org/2/library/functions.html#sorted">sorted</a> und
+        <a href="http://docs.python.org/2/library/functions.html#abs">abs</a>.
+<pre class="brush: python">
+abs(5) == abs(-5) == 5
+sorted([1, 5, 2, -3]) == [-3, 1, 2, 5]
+    </pre>
+    </div>
+</div>
+<div class="question-answer">
+    <div class="question">
+        Ich benötige etwas Hilfe, um mit der Mission weiterfahren zu können
+    </div>
+    <div class="answer">
+        Versuche <a href="https://wiki.python.org/moin/HowTo/Sorting/#Key_Functions">den Schlüssel-Parameter "key" für das Sortieren</a>.
```

Comment:
this sentence is :shit:

let's try: _Benutze den <a href="https://wiki.python.org/moin/HowTo/Sorting/#Key_Functions">Schlüsselparameter</a> (key) für das Sortieren._